src/runtime/linux.rs

```diff
@@ -573,21 +570,6 @@ make_pub!(
         #[cfg_attr(feature = "builder", getset(get = "pub"))]
         /// Unified resources.
         unified: Option<HashMap<String, String>>,
-
-        // TODO: I am not part of the official spec
-        #[serde(default)]
-        #[cfg_attr(feature = "builder", getset(get_copy = "pub"))]
-        // Disables the OOM killer for out of memory conditions
-        disable_oom_killer: bool,
-
-        #[cfg_attr(feature = "builder", getset(get_copy = "pub"))]
-        // TODO: I am not part of the official spec
-        // Specify an oom_score_adj for container
-        oom_score_adj: Option<i32>,
-
-        #[cfg_attr(feature = "builder", getset(get_copy = "pub"))]
-        // TODO: I am not part of the official spec
-        freezer: Option<FreezerState>,
```

Comment:
Do we wanna remove the FreezerState type too?

Reply:
Yes, this should have been removed. The runtime spec does not say anything about the freezer.